app/assets/stylesheets/asset-nav-bar.scss

```diff
@@ -0,0 +1,54 @@
+.asset-nav-bar li a {
+    display: block;
+    color: #fff;
+    padding: 8px 0 8px 16px;
+    text-decoration: none;
+    margin-left: -20%;
+}
+
+.asset-nav-bar li a:hover {
+    background-color: #555;
+    color: white;
+}
+
+.asset-nav-bar ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    width: 25%;
+    background-color: #f1f1f1;
+    height: 100%; /* Full height */
+    position: fixed; /* Make it stick, even on scroll */
+    overflow: auto; /* Enable scrolling if the sidenav has too much content */
+}
+
+.asset-nav {
+  width: 10%;
+  height: 100%;
+  position: relative;
+  top: 20px;
+  bottom: 0;
+  z-index: 1000;
+  background-color: #222;
+  margin-top: -40px;
```

Comment:
See if you can avoid negative margins, It shows that you are compensating for something else. However, sometimes they are necessary. But try avoid them.